packages/core-v2-sdk/package.json

```diff
@@ -47,6 +47,7 @@
   "dependencies": {
     "@elementfi/base": "*",
     "@elementfi/core-v2-typechain": "*",
+    "bignumber.js": "^9.1.0",
     "evm-bn": "^1.1.2",
```

Comment:
Do we still need `evm-bn` now that we have `bignumber.js`?

Reply:
evm-bn is just two util functions that makes it easy to go from BigNumber <-> string fixed point number, so when we read/write to the contracts. but when we are simulating logic (trades, apy, etc) we can use this library for higher precision arithmetic.